src/main/controller/UserController.ts

```diff
@@ -116,6 +120,24 @@ export class UserController {
     }
   };
 
+  getDetailedWorkout = async (req: Request, res: Response): Promise<any> => {
+    try {
+      const workout = await db.Workout.findOne({
+        attribute: ['id'],
+        where: { id: req.params.workoutId, userId: req.params.userId },
+        include: {
+          model: db.Exercise,
+          as: 'exercises',
+          through: { attributes: [] },
+        },
```

Comment:
Missing the exercise tags, but also conflicts with my endpoint,. Maybe remove this before you merge?